SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,60 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2025-01-15T01:00:00.000Z'
+  project-url: 'https://github.com/kyverno/kyverno'
+  license: 'https://github.com/kyverno/kyverno/blob/main/LICENSE'
+project-lifecycle:
+  bug-fixes-only: false
+  core-maintainers:
+    - https://github.com/kyverno/kyverno/blob/main/MAINTAINERS.md
+  status: active
+  release-process: 'https://github.com/kyverno/kyverno/releases'
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+  automated-tools-list:
+    - automated-tool: dependabot
+      action: allowed
+      path:
+        - /
+  contributing-policy: 'https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md'
+  code-of-conduct:
+  - 'https://github.com/kyverno/kyverno/blob/main/CODE_OF_CONDUCT.md'
+documentation:
+  - 'https://kyverno.io/docs/'
+distribution-points:
+  - 'https://github.com/orgs/kyverno/packages'
+security-artifacts:
+  threat-model:
+    threat-model-created: true
+    evidence-url:
+    - 'https://kyverno.io/docs/security/#threat-model'
+  self-assessment:
+    self-assessment-created: false
+security-testing:
+- tool-type: sca
+  tool-name: Dependabot
+  tool-version: "2"
+  tool-url: https://github.com/dependabot
+  integration:
+    ad-hoc: false
+    ci: true
+    before-release: true 
+security-contacts:
+- type: email
+  value: kyverno-security@googlegroups.com
+  primary: true
+vulnerability-reporting:
+  accepts-vulnerability-reports: true
+  email-contact: kyverno-security@googlegroups.com
+  security-policy: 'https://kyverno.io/docs/security/'
+  bug-bounty-available: true
+  bug-bounty-url: 'https://hackerone.com/kubernetes'
```

Comment:
I am not sure if Kyverno is covered under the Kubernetes bug bounty. Do you have an example of other CNCF incubating projects that do the same?